api/spec/openapi/ParallelCluster.openapi.yaml

```diff
@@ -697,6 +697,259 @@ paths:
+          schema:
+            type: number
+            description: The maximum number of log events returned. If you don't specify a value, the maximum is as many log events as can fit in a response size of 1 MB, up to 10,000 log events.
+            format: int32
+            nullable: true
+        - name: startTime
+          in: query
+          description: The start of the time range, expressed in ISO8601 format (e.g. '2021-01-01T20:00:00.000Z'). Events with a timestamp equal to this time or later than this time are included.
+          schema:
+            type: string
+            description: The start of the time range, expressed in ISO8601 format (e.g. '2021-01-01T20:00:00.000Z'). Events with a timestamp equal to this time or later than this time are included.
+            format: date-time
+        - name: endTime
+          in: query
+          description: The end of the time range, expressed in ISO8601 format (e.g. '2021-01-01T20:00:00.000Z'). Events with a timestamp equal to or later than this time are not included.
+          schema:
+            type: string
+            description: The end of the time range, expressed in ISO8601 format (e.g. '2021-01-01T20:00:00.000Z'). Events with a timestamp equal to or later than this time are not included.
+            format: date-time
+      responses:
+        "200":
+          description: GetClusterLogEvents 200 response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetClusterLogEventsResponseContent'
+        "400":
+          description: BadRequestException 400 response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestExceptionResponseContent'
+        "401":
+          description: UnauthorizedClientError 401 response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedClientErrorResponseContent'
+        "429":
+          description: LimitExceededException 429 response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LimitExceededExceptionResponseContent'
+        "500":
+          description: InternalServiceException 500 response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalServiceExceptionResponseContent'
+      tags:
+        - Cluster Logs
+      x-amazon-apigateway-integration:
+        credentials:
+          Fn::Sub: ${APIGatewayExecutionRole.Arn}
+        httpMethod: POST
+        payloadFormatVersion: "2.0"
+        type: aws_proxy
+        uri:
+          Fn::Sub: arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${ParallelClusterFunction.Arn}/invocations
+  /v3/clusters/{clusterName}/stackevents:
```

Comment:
In future we can have more than 1 stack. How can we list the events of substacks?
Should we expose a stacks and stacks/{stackName}/events ?